packages/kusama/src/__snapshots__/peopleKusama.proxy.e2e.test.ts.snap

```diff
@@ -321,8 +321,6 @@ exports[`People Kusama Proxy > perform proxy call on behalf of delegator > event
 
 exports[`People Kusama Proxy > proxy announcement lifecycle test > events when Alice rejects Bob's proxy call announcement 1`] = `[]`;
 
-exports[`People Kusama Proxy > proxy announcement lifecycle test > events when Alice rejects Bob's proxy call announcement 2`] = `[]`;
```

Comment:
this is most likely due to retry run on local causing extra snapshot, in that case, this is good